scanpipe/templates/scanpipe/project_list.html

```diff
@@ -21,7 +21,27 @@
             </a>
           {% endif %}
         </div>
-        <a href="{% url 'project_add' %}" class="button is-link">New Project</a>
+        <div>
+          <div class="dropdown is-right is-hoverable">
+            <div class="dropdown-trigger">
+              <button class="button" aria-haspopup="true" aria-controls="dropdown-menu">
+                <span>Sort By</span>
+                <span class="icon is-small">
+                  <i class="fas fa-angle-down" aria-hidden="true"></i>
+                </span>
+              </button>
+            </div>
+            <div class="dropdown-menu" id="dropdown-menu" role="menu">
+              <div class="dropdown-content">
+                <a href="?sort=name" class="dropdown-item">Name (A-Z)</a>
+                <a href="?sort=-name" class="dropdown-item">Name (Z-A)</a>
+                <a href="?sort=created_date" class="dropdown-item">Date Created (0-9)</a>
```

Comment:
Let's use "Newest" and "Oldest" as label.
Also, those should be first in the list as "Newest" is the default ordering.

Reply:
so dropdown will replace `Date created` with `Oldest`, `Newest`. 
what about sorting based on Names?

Reply:
Name options are fine.